### PR TITLE
Refresh ansible role

### PR DESCRIPTION
**Pull request by gwarf**
Refresh ansible role. Add required release parameter to examples, it fails without it. Update igtf default values, drop using admin-repo.egu.eu  to get CAs.

### CODE REVIEW

**Comment by gwarf** [comment]
@enolfc super-linter is not configured and more fixes will have to be done later, but I'm trying to revive this repository.
The changes I pushed have been tested locally and are required and I will need to push them to ansible-galaxy (update the org there too), can you please do a light review of this?

File: README.md [hunk 1]
```diff
@@ -1,75 +1,89 @@
...
 
-[![Build Status](https://travis-ci.org/EGI-Foundation/ansible-role-umd.svg?branch=master)](https://travis-ci.org/EGI-Foundation/ansible-role-umd) [![Docker Repository on Quay](https://quay.io/repository/egi/umd4/status "Docker Repository on Quay")](https://quay.io/repository/egi/umd4)
+[![Docker Repository on Quay](https://quay.io/repository/egi/umd4/status "Docker Repository on Quay")](https://quay.io/repository/egi/umd4)
 
-The role deploys the repository files needed to access the products
-distributed by UMD, currently supported for Scientific Linux 6 and CEntOS7.
-This role optionally deploys the Interoperable Global Trust Federation (IGTF) repository file.
+The role deploys the repository files needed to access the products distributed
+by [UMD](https://go.egi.eu/umd), currently supported for Scientific Linux 6 and
+CEntOS7. This role optionally deploys the Interoperable Global Trust Federation
```
Comment (enolfc) [inline README.md:7]:
```suggestion
CentOS7. This role optionally deploys the Interoperable Global Trust Federation
```

File: AUTHORS.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Maintainers
+-----------
+Enol Fernandez <enol.fernandez@egi.eu>
```
Comment (enolfc) [inline AUTHORS.md:3]: am I a maintainer?

**Review by enolfc — CHANGES_REQUESTED** [review]
just nitpicking

File: AUTHORS.md (hunk 2 above)
Comment (gwarf) [reply AUTHORS.md:3]: I don't wan to be alone, and you are currently reviewing a PR :)

**Review by enolfc — APPROVED** [review]
👍

File: AUTHORS.md (hunk 2 above)
Comment (enolfc) [reply AUTHORS.md:3]: let's keep it them

**Pull request merged**